conf/application.conf

```diff
@@ -186,6 +186,7 @@ limits {
   maxCountriesOfRouting = 99
   maxOfficesOfTransit = 9
   maxOfficesOfExit = 9
+  maxActiveBorderTransports = 9
```

Comment:
Can we also set this in test.application.conf? Not so bad when it's 9 but for larger numbers it can be quite exhaustive in the tests